css/basket.css

```diff
@@ -59,4 +68,62 @@
 
 .basket.basket--empty .basket__empty-text {
   display: block;
-}
\ No newline at end of file
+}
+
+.basket__mobile {
+  margin-bottom: 2rem;
+  display: flex;
+  justify-content: space-between;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  background: rgba(255, 255, 255, 0.9);
+  box-shadow: 0px 10px 16px rgba(0, 0, 0, 0.1);
+  border: 1px solid rgba(0, 0, 0, 0.1);
+  border-radius: 0.5rem;
+  padding: 0.5rem;
+}
+.basket__mobile-price {
+  font-size: 1.5rem;
+  font-weight: 800;
+  display: flex;
+  align-items: center;
+  padding: 0 0.5rem;
+}
+.basket__mobile-quantity {
+  display: flex;
+  align-items: center;
+}
+
+@media (min-width: 1024px) {
+  .basket {
+    display: block;
+    position: fixed;
+    top: 97px;
+    right: 0;
+    left: unset;
+    bottom: unset;
+    background: rgba(255, 255, 255, 0.6);
+    box-shadow: 0px 10px 16px rgba(0, 0, 0, 0.1);
+    border: 1px solid rgba(0, 0, 0, 0.1);
+    border-radius: 0.5rem;
+    width: calc(30% - 3rem);
+    padding: 1rem;
+  }
+
+  .basket__list {
+    max-height: calc(100vh - 262px);
+  }
+  /* .basket--open {
```

Comment:
czy to powinno być zakomentowane czy jednak nie? Sprawdź proszę wszędzie :)

Reply:
Poprawione :)